release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3484,10 +3484,10 @@ $ oc adm release info 4.16.33 --pullspecs
 
 * Previously, if you specified a `forceSelinuxRelabel` field in the `ClusterResourceOverride` CR and then modified the CR at a later stage, the Cluster Resource Override Operator did not apply the update to the associated `ConfigMap` resource. This `ConfigMap` resource is important for an SELinux relabeling feature, `forceSelinuxRelabel`. With this release, the Cluster Resource Override Operator now applies and tracks any `ClusterResourceOverride` CR changes to the `ConfigMap` resource. (link:https://issues.redhat.com/browse/OCPBUGS-48690[*OCPBUGS-48690*])
 
-* Previously, after you deleted a pod from an {product-title} cluster, the crun container runtime failed to stop any running containers that existed in the pod. This caused the pod to remain in a `terminating` state. With this release, a fix ensures that if you delete a pod, crun stops any running containers without placing the pod in a permanent "terminating" state. (link:https://issues.redhat.com/browse/OCPBUGS-48564[*OCPBUGS-48564*]) 
+* Previously, after you deleted a pod from an {product-title} cluster, the crun container runtime failed to stop any running containers that existed in the pod. This caused the pod to remain in a `terminating` state. With this release, a fix ensures that if you delete a pod, crun stops any running containers without placing the pod in a permanent "terminating" state. (link:https://issues.redhat.com/browse/OCPBUGS-48564[*OCPBUGS-48564*])
```

Comment:
For reviewers, it looks like there is trailing whitespace that the diff isn't highlighting, but you can see the little dot if you look close.